.github/workflows/lint_test_build.yml

```diff
@@ -1,155 +0,0 @@
-    needs: checkout
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-
-      - name: generate backend graphql
-        run: pnpm --filter backend generate
-      - name: generate auth graphql
-        run: pnpm --filter auth generate
-
-      - name: verify changed files
-        uses: tj-actions/verify-changed-files@v19
-        id: verify-changed-files
-        with:
-          files: |
-            **/generated/graphql.*
-
-      - name: fail if GraphQL was generated
-        if: steps.verify-changed-files.outputs.files_changed == 'true'
-        run: exit 1
-
-  backend:
-    runs-on: ubuntu-22.04
-    needs: [checkout]
-    timeout-minutes: 25
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter backend build:deps
-      - run: NODE_OPTIONS=--max-old-space-size=4096 pnpm --filter backend test:ci
-      - name: AsyncAPI extension
-        run: |
-          echo "{\"extends\":[\"spectral:oas\",\"spectral:asyncapi\"]}" >> .spectral.json
-      - name: Validate Open API specs
-        run: |
-          npx @stoplight/spectral-cli lint ./packages/backend/openapi/specs/webhooks.yaml
-
-  frontend:
-    runs-on: ubuntu-22.04
-    needs: checkout
-    timeout-minutes: 5
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter frontend typecheck
-      - run: pnpm --filter frontend build
-
-  auth:
-    runs-on: ubuntu-22.04
-    needs: [checkout]
-    timeout-minutes: 5
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter auth build:deps
-      - run: pnpm --filter auth test
-      - name: AsyncAPI extension
-        run: |
-          echo "{\"extends\":[\"spectral:oas\",\"spectral:asyncapi\"]}" >> .spectral.json
-      - name: Validate Open API specs
-        run: |
-          npx @stoplight/spectral-cli lint ./packages/auth/openapi/specs/id-provider.yaml
-
-  mock-account-servicing-entity:
-    runs-on: ubuntu-22.04
-    needs: checkout
-    timeout-minutes: 5
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter mock-account-servicing-entity build
-      - run: pnpm --filter mock-account-servicing-entity typecheck
-
-  token-introspection:
-    runs-on: ubuntu-22.04
-    needs: checkout
-    timeout-minutes: 5
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter token-introspection test
-      - name: AsyncAPI extension
-        run: |
-          echo "{\"extends\":[\"spectral:oas\",\"spectral:asyncapi\"]}" >> .spectral.json
-      - name: Validate Open API specs
-        run: |
-          npx @stoplight/spectral-cli lint ./packages/token-introspection/openapi/specs/token-introspection.yaml
-
-  integration-test:
```

Comment:
Looks like this step for `integration-tests` should also get moved over to the `node-build` workflow

Reply:
Nice catch! Will add it tomorrow.

Reply:
Done